modules/external-auth-disabling.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-disabling_{context}"]
+= Disabling direct authentication
+
+If necessary, you can disable direct authentication for your cluster and revert back to authenticating with the built-in OpenShift OAuth server.
+
+.Prerequisites
+
+* You have access to the `kubeconfig` file for the cluster.
```

Comment:
@bergerhoffer , it needs cluster admin permission too, so it's better to say this too, as your previous "You have access to the `kubeconfig` file generated by the installation program for the cluster" means.

Reply:
Replaced it with the wording from the other procedure